### ISSUE TITLE
WGAN-GP Loss

### ISSUE THREAD

**Issue by Mohammad-Rahmdel** [timsainb/tensorflow2-generative-models#13]
Hi,
In Wasserstein GAN with Gradient Penalty (WGAN-GP) code, I think there is a mistake in disc_loss formulation. 
disc_loss = (
            tf.reduce_mean(logits_x)
            - tf.reduce_mean(logits_x_gen)
            + d_regularizer * self.gradient_penalty_weight
        )
According to the paper disc_loss has to be D(x_tilda) - D(x) + ...
So the sign of the first two terms should be vice versa.

**Comment by tkoch96** [comment]
I think the code author just decided to invert the sign of the discriminator. Notice how the generator loss is just D(G(z)) instead of -D(G(z)) (as in the paper).
